06_modules_lab/01.py

```diff
@@ -1,4 +1,6 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times
-"""
+import sys
+index = 0
+max = int(sys.argv[1])
+while index < max:
+	print "Hello Python"
+	index = index + 1
```

Comment:
Nice, but a few comments:

* ```max``` is a built-in keyword in Python, better use another name
* instead of counting yourself, you could use ```range()```